scrapeGithub/inserUrls.py

```diff
@@ -0,0 +1,22 @@
+from database import connectToDatabase, getAllUrlsToBeScraped, addUrlsToBeScraped, markUrlAsScraped
+import sys
+import random
+
```

Comment:
File name has missing 't'

Reply:
I noticed that long ago, still left it like that 😅